packages/react-emotion/src/index.js

```diff
@@ -140,6 +140,19 @@ const createStyled = (
     Styled.__emotion_real = Styled
     Styled[TARGET_KEY] = stableClassName
 
+    Object.defineProperty(Styled, 'toString', {
+      enumerable: false,
+      value() {
+        if (
+          process.env.NODE_ENV !== 'production' &&
+          stableClassName === undefined
+        ) {
+          throw new Error('')
```

Comment:
Could you add make this error be 'Component selectors can only be used in conjunction with babel-plugin-emotion.'